components/Feed/ChannelFilter.tsx

```diff
@@ -1,43 +1,64 @@
 'use client';
 
-import { RadioGroup, RadioGroupItem } from '@/components/ui/radio-group';
-import { Label } from '@/components/ui/label';
 import { useFeedProvider } from '@/providers/FeedProvider';
+import { Button } from '@/components/ui/button';
+import { cn } from '@/lib/utils';
+import Image from 'next/image';
 
 const channels = [
-  { label: '/sonata', value: 'sonata' },
-  // { label: '/music', value: 'music' },
-  { label: '/soundscapes', value: 'soundscapes' },
-  { label: '/albumoftheday', value: 'albumoftheday' },
-  { label: '/djs', value: 'djs' },
-  { label: '/soundxyz', value: 'soundxyz' },
-  // { label: '/coop-recs', value: 'coop-recs' },
-  { label: '/bangers', value: 'bangers' },
-  // { label: '/tropicalhouse', value: 'tropicalhouse' },
+  {
+    label: '/sonata',
+    value: 'sonata',
+    icon: 'https://imagedelivery.net/BXluQx4ige9GuW0Ia56BHw/4e85acaa-4f1d-444b-1e35-dd06d43d0800/rectcrop3',
+  },
+  // { label: '/music', value: 'music', icon: 'https://ipfs.decentralized-content.com/ipfs/bafybeibdk7mvrhmud76ye6wm623sjgtiashik2imee7dkeliiq4wfissqq' },
```

Comment:
remove the unused code 🚮 

whenever we get around to adding these channels, we'll add it back